cmd/cmd.go

```diff
@@ -529,11 +529,27 @@ func generate(cmd *cobra.Command, model, prompt string, wordWrap bool, format st
 	}
```

Comment:
We should handle another case: `ollama create`

This should:1
1. Check the model in the `FROM` line to see if it's compatible
2. If the unsupposed model error is received, try to pull it if it's a `library/` model
3. If that fails (e.g. `FROM mario`), let the user know this model is based on a model incompatible with Ollama

This will give folks a chance to re-create their models with updated `llama2`, etc.

For models that are based on custom models (#3 above), it seems the best case is just to error for now

Reply:
In the release notes I'll include notes on how to re-build models from the base library models

Reply:
This case is handled now. Here's some examples:

GGML library model:
```
ollama create mario -f ~/models/mario/Modelfile
transferring model data
reading model metadata
updating base model
pulling manifest
pulling 22f7f8ef5f4c... 100% ▕███████████████████████████████████████▏(3.8 GB/3.8 GB)
pulling 8c17c2ebb0ea... 100% ▕███████████████████████████████████████▏(7.0 KB/7.0 KB)
pulling 7c23fb36d801... 100% ▕███████████████████████████████████████▏(4.8 KB/4.8 KB)
pulling 2e0493f67d0c... 100% ▕███████████████████████████████████████▏(59 B/59 B)
pulling 2759286baa87... 100% ▕███████████████████████████████████████▏(105 B/105 B)
pulling 5407e3188df9... 100% ▕███████████████████████████████████████▏(529 B/529 B)
verifying sha256 digest
writing manifest
removing any unused layers
success
... etc
```

Custom model:
```
ollama create orca-ggml -f ~/models/orca-ggml/Modelfile
transferring model data
creating model layer
Error: model binary specified in FROM field is not a valid gguf format model, unsupported model format
```